qna/DE-shop-clearing-entitlement.md

```diff
@@ -0,0 +1,13 @@
+## Question
+
+Was ist der Unterschied zwischen einem Entitlement und dem eigentlichen Produkt?
+
+## Metadata tags
+
+lang-de, market-de, shop, PosCreator, PosDealer, Consultant
+
+## Answer
+
+Im fiskaltrust.Portal finden Sie unter <kbd>Shop</kbd>  &rarr;  <kbd>Produkte</kbd> die Auswahl der fiskaltrust. Das sind neben dem Basis-Produkt, der fiskaltrust.Middleware, kostenpflichtige Addon-Produkte, wie z.B. das POS-Archiv, die Finanzamtsmeldung, technische Sicherheitseinrichtungen (TSE), das Kassenarchiv Online (AKO) sowie fiskaltrust. Sorglos-Bundles, siehe auch fiskaltrust.Docs.
+Der KassenHändler verkauft die fiskaltrust.Produkte auf dem ihm üblichen Weg. 
```

Comment:
Please clarify with Tom if we are now writing KassenHändler or Kassenhändler. (same for KassenBetreiber and KassenHersteller)